maintainers/maintainer-list.nix

```diff
@@ -6385,6 +6385,12 @@
     githubId = 364510;
     name = "Tobias Geerinckx-Rice";
   };
+  ndl = {
```

Comment:
Please put this into a separate commit with the message ``maintainers: add ndl``.

Reply:
Done.